sdk/kfp-tekton-limitation.md

```diff
@@ -0,0 +1,9 @@
+
+## kfp-tekton limitation 
+- Please be aware that defined Affinity, Node Selector, and Tolerations are applied to all the tasks in the same pipeline because there's only one podTemplate allowed in each pipeline.
+
+- When you add test cases to compiler_tests, the output of pipeline/pipelinerun yaml may has uncertain values or orders, then you can define a lambda function as normalize_compiler_output_function to pass the testing.
+
+- When you run kfp-tekton pipeline with sidecars, you may notice a completed pod and a sidercar pod with Error. This is a known limitation from tekton pipeline with sidecar output. [Tekton pipeline readme](https://github.com/tektoncd/pipeline/blob/master/docs/developers/README.md#handling-of-injected-sidecars) has documented this limitation, and the [tracking issue](https://github.com/tektoncd/pipeline/issues/1347). It has no function impacts.
```

Comment:
Can we add a little bit more details to this such as the comments in https://github.com/kubeflow/kfp-tekton/issues/44?

Reply:
ok